src/test/java/com/github/rodionovsasha/commentservice/services/UserServiceTest.java

```diff
@@ -0,0 +1,126 @@
+package com.github.rodionovsasha.commentservice.services;
+
+import com.github.rodionovsasha.commentservice.config.TestConfig;
+import com.github.rodionovsasha.commentservice.exceptions.InactiveUserException;
+import com.github.rodionovsasha.commentservice.exceptions.UserNotFoundException;
+import lombok.val;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = TestConfig.class)
+@TestPropertySource(locations = "classpath:test.properties")
+@SpringBootTest
+@Transactional
+public class UserServiceTest {
```

Comment:
Is it the _old_ test? Should we delete it?

Reply:
forgot to push :)